demo/pkg/subgraphs/employees/subgraph/schema.graphqls

```diff
@@ -1,142 +1,143 @@
 directive @goField(
-    forceResolver: Boolean
```

Comment:
Just formatting or did you change anything?

Reply:
New query added and implemented, `firstEmployee`.